### DISCUSSION TITLE
How to set project root for quasar paths (lost after upgrade to 1.15.16)

### DISCUSSION THREAD

**Discussion by apavel3458** [quasarframework/quasar#9407] [General - Components / Directives / etc] [answered]
Hi there, 

I recently upgraded to quasar 1.15.16, and lost project root. None of the HTML images and favicons load correctly, i have to change everything to relative paths.   How do we fix this? this seems to be a webpack setting. 

Thanks!

**Comment by hawkeye64** [comment, marked as answer]
You're @quasar/app (cli) probably updated and now you need to follow the upgrade path: https://quasar.dev/quasar-cli/app-upgrade-guide#upgrade-guide

**Reply by apavel3458** [reply]
Thank you! I went through the instructions and it worked.  Have  a great day!
